components/network.py

```diff
@@ -528,6 +528,9 @@ def _process_queue(self):
                 packet_drop_var = random.random()
                 if packet_drop_var > (1 - self.packet_drop_rate):
                     Logger.get_instance().log("PACKET DROPPED")
+                    if packet.payload_type == protocols.QUANTUM:
+                        packet.payload.release()
+                        print("Qubit was released")
```

Comment:
Better to log this message than print it

Reply:
deleted it completely, since the package drop is already logged.